scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -2,69 +2,30 @@
 
 # fail if some commands fails
 set -e
-# show commands
-set -x
-
-ARCH=$(uname -m)
-export CI="openshift"
-if [ "${ARCH}" == "s390x" ]; then
-    make configure-installer-tests-cluster-s390x
-elif [ "${ARCH}" == "ppc64le" ]; then
-    make configure-installer-tests-cluster-ppc64le
-else
-    make configure-installer-tests-cluster
+# Hide command that deals with secrets
+set +x
+if [[ -f $ODO_RABBITMQ_AMQP_URL ]]; then
+    export AMQP_URI=$(cat $ODO_RABBITMQ_AMQP_URL)
```

Comment:
Does AMQP_URI contains the url to the instance of rabbitMQ? and why do we need to export AMQP_URI are we using this Env variable outside  `scripts/openshiftci-presubmit-all-tests.sh` ?